### ISSUE TITLE
W3C markup validation error in slider template

### ISSUE THREAD

**Issue by aadmathijssen** [EmicoEcommerce/Magento1Tweakwise#18]
Issue Brief
--
The [W3C markup validation service](https://validator.w3.org/) reports the following issue on the HTML generated by the [`app/design/frontend/base/default/template/emico_tweakwise/catalog/layer/facet/slider.phtml`](https://github.com/EmicoEcommerce/Magento1Tweakwise/blob/v1.11.5/app/design/frontend/base/default/template/emico_tweakwise/catalog/layer/facet/slider.phtml#L32) template:

```
Error: Element div not allowed as child of element ol in this context.
```

Environment
--
- [x] PHP Version: 7.0
- [x] Magento Version: 1.9.3.8
- [x] Magento Enterprise: no
- [x] Tweakwise Export Version: 1.11.5
- [x] Magento Deploy Mode: production

Steps to reproduce
--

1. On a website with the Emico_Tweakwise plugin installed and configured such that layered navigation is enabled, navigate to a category page that shows a price slider (this should render the `app/design/frontend/base/default/template/emico_tweakwise/catalog/layer/facet/slider.phtml` template).
2. Go to https://validator.w3.org/, click on "Validate by direct upload", paste the contents of the generated HTML, and click the "Check" button.

Actual result
--

The W3C markup validator service reports the following error:

```
Error: Element div not allowed as child of element ol in this context. (Suppressing further errors from this subtree.)

From line 1263, column 13; to line 1270, column 17

          <div class="slider-container"↩                 data-url-key="price"↩                 data-min-value="69"↩                 data-max-value="3356.58"↩                 data-lower-value="69"↩                 data-upper-value="3356.58"↩                >↩     

Contexts in which element div may be used:
    Where flow content is expected. 
    As a child of a dl element. 
Content model for element ol:
    Zero or more li and script-supporting elements. 
```

Expected result
--

The W3C markup validation service does not show an error.

**Comment by edwinljacobs** [comment]
We removed the ol, it does not seem necessary to have that there. Will be released soon

**Comment by edwinljacobs** [comment]
This has been released in v1.11.13
